.github/workflows/build.yaml

```
@@ -46,14 +46,18 @@ jobs:
         with:
           apt-get: git python3-jinja2 python3-yaml python3-deepdiff python3-git python3-github python3-requests xmldiff
       # TODO: Use action's checkout along with --local and --repository options of ctf
+      # See: https://github.community/t/how-to-commit-to-two-branches-from-an-action/17713/4
       #- name: Checkout
-      #  uses: actions/checkout@v2
+      #  uses: actions/checkout@v1
       - name: Checkout (CTF)
         uses: actions/checkout@v2
         with:
           repository: mildas/content-test-filtering
           path: ctf
-      - name: Process
-        run: python3 ./ctf/content_test_filtering.py pr --output-format markdown ${{ github.event.pull_request.number }} > ctf.md
-      - name: Update the PR
-        run: python3 ./ctf/utility_scripts/comment_pr.py --token ${{ github.token }} --pr ${{ github.event.pull_request.number }} --comment ctf.md
+      - name: Process (see the output for recommended tests)
+        run: python3 ./ctf/content_test_filtering.py pr --output-format markdown ${{ github.event.pull_request.number }} # > ctf.md
```

I can't see the output in the logs of this line. I think you have to explicitly echo the output, something like this:

```suggestion
        run: python3 ./ctf/content_test_filtering.py pr --output-format markdown ${{ github.event.pull_request.number }} # > ctf.md
        id: ctf
      - name: CTF Output
        run: echo ${{ steps.ctf.outputs.stdout }}
```

---

As far as I know the tool won't output anything if there are no relevant changes detected. The change you proposed could be used to carry some output over to the next step (which you created), but it is not needed in general.

---

It makes sense. Maybe the CTF can be improved in this case to print at least some message that there is nothing to be tested, otherwise it can confuse people because it says look for the output but there is nothing. But I'm fine with this approach